pkg/cli/rook_wait_for_version.go

```diff
@@ -36,7 +36,7 @@ func NewRookWaitForRookVersionCmd(cli CLI) *cobra.Command {
 }
 func NewRookWaitForCephVersionCmd(cli CLI) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "wait-for-ceph-version",
+		Use:   "wait-for-ceph-version version",
```

Comment:
```suggestion
		Use:   "wait-for-ceph-version [version]",
```

Reply:
it's not optional, so no brackets

Reply:
Ah. I didn’t realize that is what brackets meant. Then can you make it all caps to indicate it’s an argument.

Reply:
I don't think that's the recommended usage: https://pkg.go.dev/github.com/spf13/cobra@v1.5.0#Command.Use

Reply:
docker and kubectl use this pattern

```
$ kubectl label --help

Usage:
  kubectl label [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ...
KEY_N=VAL_N [--resource-version=version] [options]
```

```
$ docker inspect --help

Usage:  docker inspect [OPTIONS] NAME|ID [NAME|ID...]
```